src/restapi/mm_routes.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import express, {Request, Response} from 'express';
+import {AppCall} from 'mattermost-redux/types/apps';
+
+import app from '../app/app';
+
+import {jsonStoreFileName} from '../app/constants';
+
+const router = express.Router();
+
+router.post('/createform', async (req: Request, res: Response) => {
+    const appCall: AppCall = req.body;
+    if (appCall.type === 'form') {
+        const createForm = app.calls.createForm(appCall.context.post.message);
+        res.json(createForm);
+    } else {
+        const message = app.createTicketFromPost(appCall);
+        res.json({});
```

Comment:
These API handlers should be wrapped in a `try/catch` block, otherwise the HTTP request would not be closed when an error occurs, as far as I know.

Reply:
https://github.com/mattermost/mattermost-app-zendesk/commit/09732405a9469420730de8af14c8a1ba9d1164e1